### ISSUE TITLE
Keyboard font: "Tux" ligature not working

### ISSUE THREAD

**Issue by Moonbase59** [alerque/libertinus#415]
**Describe the bug**
From the sources, I expected the "Tux" ligature to produce a "Tux" key (instead of the "Windows" key).
This doesn’t work with the `LibertinusKeyboard-Regular.otf` (v7.0.20) font.

There _are_ actually keyboards that have the tux instead of the Windows, so it’d be nice if this worked.

**Steps to reproduce**
Install `LibertinusKeyboard-Regular.otf`, open a new LibreOffice Writer document, set font to `Libertinus Keyboard` and type "Tux". Three keys T, u, x are generated, instead of one Tux key.

**Expected vs. actual behavior**
See above.

**Comment by alerque** [comment]
Sure. Happy to accept a contribution of a new Tux glyph.

**Comment by alerque** [comment]
Just a heads up, I poked around and there is a GNU glyph ligature if that tickles your fancy. An Apple command key glyph might not be a terrible follow up addition to a Tux.

**Comment by Moonbase59** [comment]
Unfortunately I have no idea how to do that, just remembered that "Linux Biolinum Keyboard" has a Tux, uses many of the same ligatures, and in general looks much like "Libertinus Keyboard", maybe it was originally forked from "Linux Biolinum Keyboard"? If the licenses allow, maybe the Tux could be copied from it?

I still prefer Libertinus Keyboard, since it has the ligatures for the German variants, like "Strg", "AltGr", "Bildup", "Bilddown", "Pos1" and so forth.

Thanks for taking the project over and caring about it so much!

**Comment by alerque** [comment]
Yes, this was forked from *Linux Biolinum Keyboard* — and yes the licenses are compatible. It shouldn't be too hard to port across if the glyph was there. I don't know why it would have gotten dropped along the line but I'll look into it.

**Comment by Moonbase59** [comment]
Thanks for confirming—and great that the licenses are compatible! I much prefer the "Libertinus" font family for professional typesetting (usually with LaTeX).

I find that some of the "Linux Libertine" fonts have apparently been updated since whenever they’ve been forked, especially (also) the keyboard font. Let me address some proposed updates in a new issue (and probably close this one?).

**Comment by Moonbase59** [comment]
See https://github.com/alerque/libertinus/issues/416

**Comment by LochanLoch** [comment]
GNU and Tux I have now :-) thanks guys - but DANTE - the Lion?
A list for "Libertinus Keyboard" like
"Strg" "Shift" and so on would be very helpful. If exist I don't found it.
only a help for Lyx "another-way-to-represent-keys-in-lyx" and the way over "uniE104" Alt+C behind U+e104 work only for "Linux Biolinum Keyboard" under LibreOffice the font I Installed should have it (FontForge) but I think the problem sitting still in front of pc ... and that's me. [list I building now with help over FontForge]
